src/cclib/parser/daltonparser.py

```diff
@@ -501,6 +515,9 @@ def extract(self, inputfile, line):
             self.set_attribute("charge", int(line.split()[-1]))
         if "@    Spin multiplicity and 2 M_S" in line:
             self.set_attribute("mult", int(line.split()[-2]))
+        # Dalton only has ROHF, no UHF
+            if self.mult != 1:
+                self.metadata["spintype"] = "ROHF"
```

Comment:
I'm in agreement with both @langner and @ghutchis. I think it's sufficient to have `self.metadata['methods']` as a list, where only the method and no reference to the spin restriction is made, and then have a boolean for the spin restriction. For example, `self.metadata['unrestricted'] == False` and `len(self.homos == 2)` would indicate restricted open-shell, and a small method would convert this into `"ROHF"`.